py/server/deephaven/experimental/iceberg.py

```diff
@@ -243,6 +340,95 @@ def table(self, instructions: Optional[IcebergInstructions] = None, snapshot_id:
             return IcebergTable(self.j_object.table(snapshot_id, instructions))
         return IcebergTable(self.j_object.table(instructions))
 
+    def append(self,
+               table_identifier: str,
+               tables: List[Table],
+               partition_paths: Optional[List[str]] = None,
+               instructions: Optional[IcebergParquetWriteInstructions] = None):
+        # TODO Review javadoc in this file once again
+        """
+        Append the provided Deephaven table as a new partition to the existing Iceberg table in a single snapshot. This
+        will not change the schema of the existing table.
+
+        Args:
+            table_identifier (str): the identifier string for iceberg table to append to.
+            tables (List[Table]): the tables to append.
+            partition_paths (Optional[List[str]]): the partitioning path at which data would be appended, for example,
```

Comment:
again, this is unclear.  it says "path", but the arg is a list.  Should it be "paths"?  If so, the comment is unclear about what happens with multiple tables and multiple partition paths.